webapp/channels/src/components/admin_console/team_channel_settings/group/group_list.tsx

```diff
@@ -12,57 +12,62 @@ import GroupRow from './group_row';
 
 import type {PropsFromRedux, OwnProps} from './index';
 
-const Header = () => {
-    return (
-        <div className='groups-list--header'>
-            <div className='group-name group-name-adjusted'>
+const Header = () => (
```

Comment:
Could you revert the changes to the `Header` component? They're not really needed, and they make it harder to see what's changing in this PR